GiniVision/Assets/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,117 @@
+/* 
+  Localizable.strings
+  GiniVision
+
+  Created by Peter Pult on 08/06/16.
+  Copyright © 2016 Gini. All rights reserved.
+*/
+
+"ginivision.navigationbar.camera.title"        = "Photograph document";
+"ginivision.navigationbar.onboarding.title"    = "Instructions";
+"ginivision.navigationbar.review.title"        = "Check photo quality";
+"ginivision.navigationbar.analysis.title"      = "Etwas Geduld, analysiere Foto";
+"ginivision.navigationbar.camera.close"        = "Close";
+"ginivision.navigationbar.camera.help"         = "Help";
+"ginivision.navigationbar.onboarding.continue" = "Next";
+"ginivision.navigationbar.review.back"         = "Back";
+"ginivision.navigationbar.review.close"        = "Close";
+"ginivision.navigationbar.review.continue"     = "Next";
+"ginivision.navigationbar.analysis.back"       = "Cancel";
+"ginivision.navigationbar.help.backToCamera"   = "Back";
+"ginivision.navigationbar.help.backToMenu"     = "Back";
+
+
+"ginivision.camera.captureButton" = "Capture";
+"ginivision.camera.notAuthorized" = "Permission to access the camera is required to analyse photos";
+"ginivision.camera.notAuthorizedButton" = "Grant permission";
+"ginivision.camera.captureFailed" = "Image capture failed";
+"ginivision.camera.filepicker.photoLibraryAccessDenied" = "Permission to access your photos gallery is required to analyse saved photos";
+"ginivision.camera.filepicker.mixedDocumentsUnsupported" = "It is only possible to select a file type at the same time.";
+"ginivision.camera.filepicker.errorPopup.cancelButton" = "Cancel";
+"ginivision.camera.filepicker.errorPopup.confirmButton" = "OK";
+"ginivision.camera.filepicker.errorPopup.grantAccessButton" = "Grant permission";
+"ginivision.camera.documentValidationError.general" = "Unfortunately, this file can not be edited";
+"ginivision.camera.documentValidationError.excedeedFileSize" = "Unfortunately, this file is too big";
+"ginivision.camera.documentValidationError.tooManyPages" = "The document can only have a maximum of 10 pages.";
```

Comment:
Some translations have a period at the end and some don't. It's a little inconsistent.

Reply:
It depends on where they appear, sometimes it's is needed sometimes is not.